RealdashCAN Serial Send Protocol.

celturbo opened the pull request:
Implementation of the RealdashCAN serial protocol on Speeduino serial 03. I am implementing the RealdashCAN Serial protocol on Speeduino Serial 03 with Mega 2560. -The code takes all the data and sends it every 8 bytes with a specific ID and sends it at a frequency of 200Hz. -Connection to Realdash is done as RealdashCAN. -Only some are mapped in XML, for now. -I’m planning to implement CAN Broadcast for the STM32 platform, so the same XML can be used. Anyone who can do testing will be of great help.


XML File:

[https://github.com/celturbo/RealDash-extras/tree/Speeduino-XML/RealDash-CAN/XML-files/speeduino](url)

![image](https://github.com/user-attachments/assets/ff91a518-eb6f-4139-803d-cb4e9fd72ed4)

![image](https://github.com/user-attachments/assets/36b0036e-88db-4434-a021-8a5a057684ca)

choyr commented:
> > i can test it for you
> > I would appreciate it if you could test it.

your github page does not work

celturbo committed:
RealdashCAN Serial Send Protocol.

Implementation of the RealdashCAN serial protocol on Speeduino serial 03. I am implementing the RealdashCAN Serial protocol on Speeduino Serial 03 with Mega 2560. -The code takes all the data and sends it every 8 bytes with a specific ID and sends it at a frequency of 200Hz. -Connection to Realdash is done as RealdashCAN. -Only some are mapped in XML, for now. -I’m planning to implement CAN Broadcast for the STM32 platform, so the same XML can be used. Anyone who can do testing will be of great help.

celturbo commented:
> any additional hardware required?
um bluetooth ou um USB FTDI.

celturbo committed:
update

choyr commented:
is xml updated now?
